pems/static/css/styles.css

```diff
@@ -10,3 +10,8 @@ header {
     }
   }
 }
+
+.streamlit-app {
```

Comment:
is there a class in Bootstrap already that already sets these 2 properties? if so, use that one